hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/OmTableInsightTask.java

```diff
@@ -271,14 +327,15 @@ private void handlePutEvent(OMDBUpdateEvent<String, Object> event,
                               Collection<String> sizeRelatedTables,
```

Comment:
We don't need to pass this argument as "getTablesToCalculateSize" is public method.

Reply:
Done!